### ISSUE TITLE
History sync via trusted node or another client

### ISSUE THREAD

**Issue by MAH69IK** [TokTok/c-toxcore#1160]
I use Tox on 3 devices: home laptop, work computer and telephon. Will be great if Tox can sync chat history to all devices. For example - from trusted node (I have my own node). Or if I can sync history from my friend's Tox (I send messages from work to my friend, go home, run tox and load this messages to history from friend's client).

**Comment by iphydf** [comment]
Yes, this is a good idea. I have some ideas on how to implement this (and many other features) in one go.

**Comment by zoff99** [comment]
@MAH69IK @iphydf we would need MsgV2 first, to have uniqe IDs for all messages.
otherwise it will be difficult to sync messages
